jboss/container/wildfly/launch/keycloak/2.0/module.yaml

```diff
@@ -7,46 +7,40 @@ execute:
   user: '185'
 envs:
   - name: "APPLICATION_NAME"
-    description: Application name
+    description: Optional. Used as a prefix to the client name that is computed based on the deployment name.  
   - name: "HOSTNAME_HTTP"
-    description: Custom http Route Hostname
+    description: Optional. Custom HTTP Openshift route. When not set a route discovery is operated.
   - name: "HOSTNAME_HTTPS"
-    description: Custom https Route Hostname
+    description: Optional. Custom HTTPS Openshift route. When not set a route discovery is operated.
   - name: "SSO_DISABLE_SSL_CERTIFICATE_VALIDATION"
-    description: true or false. Enable/disable validation.
+    description: Optional. true or false. Enable/disable validation of the keycloak server certificate. It is advised to be set to true when the SSO server generates a self signed certificate.
   - name: "SSO_PASSWORD"
-    description: The password for the keycloak service user.
-  - name: "SSO_PRINCIPAL_ATTRIBUTE"
-    description: Principal attribute.
-  - name: "SSO_PUBLIC_KEY"
-    description: SSO Public Key.
+    description: The password of a user that has the priviledges to interact with the keycloak realm and can create and register clients.
   - name: "SSO_REALM"
     description: The SSO realm to which the application client(s) should be associated (e.g. demo).
   - name: "SSO_SAML_CERTIFICATE_NAME"
-    description: Certificate name
+    description: Alias of the private key in the SAML client keystore.
```

Comment:
isn't it the alias of the certificate (the one EAP needs to store into RH-SSO at registration time)?